lua/entities/gmod_wire_expression2/core/matrix.lua

```diff
@@ -86,41 +84,19 @@ e2function matrix2 identity2()
 			 0, 1 }
 end
 
-/******************************************************************************/
-
-registerOperator("ass", "xm2", "xm2", function(self, args)
-	local op1, op2, scope = args[2], args[3], args[4]
-	local      rv2 = op2[1](self, op2)
-	self.Scopes[scope][op1] = rv2
-	self.Scopes[scope].vclk[op1] = true
-	return rv2
+registerOperator("is", "xm2", "n", function(state, this)
+	return (this[1] ~= 0
+		or this[2] ~= 0
+		or this[3] ~= 0)
+		and 1 or 0
 end)
 
-/******************************************************************************/
-// Comparison
-
-e2function number operator_is(matrix2 rv1)
-	if rv1[1] > delta or -rv1[1] > delta or
-	   rv1[2] > delta or -rv1[2] > delta or
-	   rv1[3] > delta or -rv1[3] > delta or
-	   rv1[4] > delta or -rv1[4] > delta
-	   then return 1 else return 0 end
-end
-
 e2function number operator==(matrix2 rv1, matrix2 rv2)
```

Comment:
where has the delta magic gone?
@Vurv78 you can't just change behaviour like this. People's chips rely on this.

Reply:
Did you read my post?

If you can give me a single person whose chip is broken by this I will delete the PR.

Reply:
it doesn't need to be all or nothing like that. just keep the existing behaviour, otherwise you're mixing an already very large refactor with breaking changes